.buildkite/pipeline-docker.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+  - label: ":pip: Install dependencies"
+    key: "pip"
```

Comment:
Consider use a separate script to kickoff the pipeline `pipeline-docker.yml`. This gives you the flexibility to control when to CI and when not to CI, based on different github PR labels, etc.

Reply:
Added a new ticket #41 for this. Will do this in v0.1.x